.changeset/@mastra_playground-ui-9850-dependencies.md

```
@@ -0,0 +1,5 @@
+---
+"@mastra/playground-ui": patch
+---
+dependencies updates:
+  - Updated dependency [`@dagrejs/dagre@^1.1.8` ↗︎](https://www.npmjs.com/package/@dagrejs/dagre/v/1.1.8) (from `^1.1.5`, in `dependencies`)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)